hacking/aws_config/testing_policies/compute-policy.json

```diff
@@ -276,5 +276,23 @@
             ],
             "Resource": "arn:aws:sqs:*:*:*"
         },
+        {
+            "Sid": "AllowKinesis",
+            "Effect": "Allow",
+            "Action": [
+                "kinesis:AddTagsToStream",
+                "kinesis:CreateStream",
+                "kinesis:DeleteStream",
+                "kinesis:DecreaseStreamRetentionPeriod",
+                "kinesis:DescribeStream",
+                "kinesis:IncreaseStreamRetentionPeriod",
```

Comment:
Looks like we're missing `kinesis:ListStreams` in the hacking policies.

Reply:
There's currently no _info module, so nothing uses it.  I'll get it added.